tasks/downloadModel3dTask.py

```diff
@@ -12,7 +12,6 @@ class DownloadModel3dTask(QgsTask):
     """QgsTask pobierania PRG"""
```

Comment:
Tak teraz zwróciłem na to uwagę, zmień proszę opis do tej funkcji na adekwatny do tego co robi ta funkcja